bundles/binding/org.openhab.binding.dmx/src/main/java/org/openhab/binding/dmx/internal/core/DmxController.java

```diff
@@ -49,9 +51,9 @@
      */
     @Override
     public void start() throws Exception {
-
         logger.trace("Starting Dmx transmitter ...");
         transmitter = new DmxTransmitter(this);
+        transmitter.setTransmitRefresh(transmitRefreshMs);
```

Comment:
Could the variables/members/methods all use the phrase `refreshInterval` so the changed code is easier to follow?